src/odb/src/db/dbPowerDomain.cpp

```diff
@@ -154,6 +155,11 @@ dbIStream& operator>>(dbIStream& stream, _dbPowerDomain& obj)
   stream >> obj._x2;
   stream >> obj._y1;
   stream >> obj._y2;
+  // User Code Begin >>
+  if (stream.getDatabase()->isSchema(db_schema_level_shifter)) {
+    stream >> obj._levelshifters;
+  }
+  // User Code End >>
```

Comment:
Could this be replaced by specifying 'schema' on the _levelshifters field in the schema?